api/tools/src/main/java/org/pmiops/workbench/tools/SetAccessModuleTimestamps.java

```diff
@@ -0,0 +1,99 @@
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.pmiops.workbench.access.AccessModuleService;
+import org.pmiops.workbench.access.AccessModuleServiceImpl;
+import org.pmiops.workbench.access.UserAccessModuleMapperImpl;
+import org.pmiops.workbench.actionaudit.ActionAuditServiceImpl;
+import org.pmiops.workbench.actionaudit.auditors.UserServiceAuditorImpl;
+import org.pmiops.workbench.audit.ActionAuditSpringConfiguration;
+import org.pmiops.workbench.config.CacheAccessModuleConfiguration;
+import org.pmiops.workbench.db.dao.UserDao;
+import org.pmiops.workbench.db.model.DbAccessModule.AccessModuleName;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@Import({
+  AccessModuleServiceImpl.class,
+  ActionAuditServiceImpl.class,
+  ActionAuditSpringConfiguration.class, // injects com.google.cloud.logging.Logging
+  CacheAccessModuleConfiguration.class, // injects List<DbAccessModule>
+  CommonMappers.class,
+  UserAccessModuleMapperImpl.class,
+  UserServiceAuditorImpl.class,
+})
+public class SetAccessModuleTimestamps {
+
+  private static final Logger log = Logger.getLogger(SetAccessModuleTimestamps.class.getName());
+
+  // very crude POC implementation - only handles ProfileConfirmation and completion (not bypass)
+  private final AccessModuleName moduleName = AccessModuleName.PROFILE_CONFIRMATION;
+  private final Timestamp timestamp = Timestamp.from(Instant.parse("2011-08-01T00:00:00.00Z"));
+
+  void applyTimestampUpdateToUser(
+      UserDao userDao,
+      AccessModuleService accessModuleService,
+      String username,
+      AccessModuleName moduleName,
+      @Nullable Timestamp timestamp,
+      boolean isBypass) {
+    DbUser dbUser = userDao.findUserByUsername(username);
+    if (dbUser == null) {
+      throw new IllegalArgumentException(String.format("User %s not found!", username));
+    }
+
+    if (moduleName == AccessModuleName.PROFILE_CONFIRMATION && !isBypass) {
+      final String logMsg =
+          String.format(
+              "Updating %s %s time for user %s to %s",
+              moduleName, isBypass ? "bypass" : "completion", username, timestamp.toString());
+      log.info(logMsg);
+
+      // dual-write to DbUser and AccessModuleService
+      // we will remove the module fields in DbUser soon
+
+      dbUser.setProfileLastConfirmedTime(timestamp);
+      dbUser = userDao.save(dbUser);
+      accessModuleService.updateCompletionTime(dbUser, moduleName, timestamp);
+    } else {
+      throw new IllegalArgumentException("Not implemented!");
+    }
+  }
+
+  private static final Option userOpt =
```

Comment:
opt nit: I would probably put static constants first above methods, except maybe in tests, though I do see the appeal of putting it close to the run cmd